pkg/connector/role.go

```diff
@@ -328,8 +329,14 @@ func (r *roleResourceType) UpdateAccountMember(ctx context.Context, accountID, m
 			ErrorLink:        endpointUrl,
 		}
 		if resp != nil {
+			bodyBytes, err := io.ReadAll(resp.Body)
+			if err != nil {
+				ce.ErrorSummary = fmt.Sprintf("Error reading response body %s", err.Error())
+				return nil, ce
```

Comment:
Similar to the other PRs, ErrorCode should be set before returning here.

Reply:
Fixed